src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -51,12 +53,13 @@
     private final String folder;
     private final String customizationSpec;
     private final boolean powerOn;
+    private final Integer timeoutInSeconds;
```

Comment:
Yeah sorry, I'm pretty new to all this code so bear with me a bit.

I like the idea of keeping everything except the constructor as int so I'll do that.